openfl/component/collaborator/collaborator.py

```diff
@@ -123,15 +132,15 @@ def __init__(self,
 
         self.task_runner.set_optimizer_treatment(self.opt_treatment.name)
 
-    def set_available_devices(self, cuda: Tuple[str] = ()):
+    def set_available_devices(self, cuda: Union[Tuple[()], Tuple[str]] = ()) -> None:
```

Comment:
What does `Tuple[()]` mean?, Looks too complex, may be simply leave `Tuple[str]`?

Reply:
I made it based on this: https://docs.python.org/3/library/typing.html#:~:text=typing.Tuple,written%20as%20Tuple%5B()%5D.

Reply:
Since `cuda` tuple  can contain an arbitrary number of arguments, I suggest simply:
```suggestion
    def set_available_devices(self, cuda: Tuple[str, ...] -> None:
```